test/unit/specs/components/Personalization/index.spec.js

```diff
@@ -10,96 +10,4 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-import { defer } from "../../../../../src/utils";
-import createPersonalization from "../../../../../src/components/Personalization";
-import createConfig from "../../../../../src/core/config/createConfig";
-import {
-  PAGE_WIDE_SCOPE_DECISIONS,
-  PAGE_WIDE_SCOPE_DECISIONS_WITHOUT_DOM_ACTION_SCHEMA_ITEMS,
-  SCOPES_FOO1_FOO2_DECISIONS
-} from "./responsesMock/eventResponses";
-
-describe("Personalization", () => {
-  const config = createConfig({ prehidingStyle: "" });
-  const logger = {
-    log() {},
-    warn() {}
-  };
-  const eventManager = {
-    createEvent() {},
-    sendEvent() {}
-  };
-
-  let event;
-  let response;
-
-  beforeEach(() => {
-    event = jasmine.createSpyObj("event", ["mergeQuery"]);
-    response = jasmine.createSpyObj("response", ["getPayloadsByType"]);
-  });
-
-  it("should return an array of decisions in lifecycle::onResponse for provided decisions decisionScopes", () => {
-    const decisionScopes = ["Foo1", "Foo3"];
-
-    response.getPayloadsByType.and.returnValue(SCOPES_FOO1_FOO2_DECISIONS);
-
-    const personalization = createPersonalization({
-      config,
-      logger,
-      eventManager
-    });
-
-    const deferred = defer();
-    const onResponse = func => {
-      const result = func({ response });
-
-      deferred.resolve(result);
-    };
-
-    personalization.lifecycle.onBeforeEvent({
-      event,
-      decisionScopes,
-      onResponse
-    });
-
-    return expectAsync(deferred.promise).toBeResolvedTo({
-      decisions: SCOPES_FOO1_FOO2_DECISIONS
-    });
-  });
-
-  it("should return an array of not rendered decisions in lifecycle::onResponse for provided decision decisionScopes and the page wide scope", () => {
-    const renderDecisions = true;
-    const decisionScopes = ["Foo1", "Foo3"];
-    const decisions = SCOPES_FOO1_FOO2_DECISIONS.concat(
-      PAGE_WIDE_SCOPE_DECISIONS
-    );
-    response.getPayloadsByType.and.returnValue(decisions);
-
-    const expectedResponse = SCOPES_FOO1_FOO2_DECISIONS.concat(
-      PAGE_WIDE_SCOPE_DECISIONS_WITHOUT_DOM_ACTION_SCHEMA_ITEMS
-    );
-    const personalization = createPersonalization({
-      config,
-      logger,
-      eventManager
-    });
-
-    const deferred = defer();
-    const onResponse = func => {
-      const result = func({ response });
-
-      deferred.resolve(result);
-    };
-
-    personalization.lifecycle.onBeforeEvent({
-      event,
-      renderDecisions,
-      decisionScopes,
-      onResponse
-    });
-
-    return expectAsync(deferred.promise).toBeResolvedTo({
-      decisions: expectedResponse
-    });
-  });
-});
+describe("Personalization", () => {});
```

Comment:
You can delete this file entirely. The script that runs that checks to see if a spec files exists for each source file ignores `index.js` files.

Reply:
ok, thanks